config.py

```diff
@@ -30,6 +30,8 @@ class Config(object):
         RAS_SECURE_MESSAGING_SERVICE_PROTOCOL,
         RAS_SECURE_MESSAGING_SERVICE_HOST, RAS_SECURE_MESSAGING_SERVICE_PORT)
 
+    RAS_SECURE_MESSAGING_SERVICE_THREAD = f"{RAS_SECURE_MESSAGING_SERVICE}v2/threads/"
```

Comment:
I don't think that this shouldn't be config; it doesn't change between environments, is a astable endpoint and isn't a secret that can't be put in the public domain.

Reply:
Still not sure why properties need to be someothing that change over enviroments. In properties you place what actually never change but might be used in different place on the coode. Search/replace string is an infamous bad practice. For me if you hav ea String that is repted more then once in ifferent code place should be placed somewhere common. How a conf/property file can be worse then search string in the code ?

Reply:
I'm not sure I understand your comment there Angello but I agree with James this doesn't need to be in the flask config the endpoint shouldn't change.

Reply:
changed